priv/repo/migrations/20220511095327_create_keyword_uploads.exs

```diff
@@ -0,0 +1,15 @@
+defmodule GoogleSearchDataViewer.Repo.Migrations.CreateKeywordUploads do
+  use Ecto.Migration
+
+  def change do
+    create table(:keyword_uploads) do
+      add :name, :string
+      add :html, :string
```

Comment:
```suggestion
      add :html, :text
```

Should we update it to text to make sure the field size large enough to content all Html content?

Reply:
Yes, it will remove the maximum length of 255? Will change that now.